### ISSUE TITLE
[RTL] arrows direction

### ISSUE THREAD

**Issue by israelidanny** [zippyui/react-date-picker#112]
Out of the box right to left support is actually not that bad on the component. One problem I noticed when I tried to use this component in an RTL environment is that the arrows end up pointing in wrong directions.

I submitted a PR that would for now allow to cope with it by simply overriding the `arrows` prop, until the component becomes officially RTL compliant. 

This is also useful for styling - for people who don't like the default arrows.

**Comment by radubrehar** [comment]
Thanks for the PR

I added the `navBarArrows` prop on the `DateField` / `MonthView` / `Calendar`

So basically you can do

```jsx
<DateField  //or MonthView or Calendar
  navBarArrows={{
     '-2': '<<',
     '2': '>>',
     '-1': '<',
    '1': '>'
  }}
/>
```
or basically you can nest a `NavBar` inside a `Calendar` or `MonthView` and use the `arrows` prop:
```jsx
<Calendar>
 <NavBar
  arrows={{
     '-2': '<<',
     '2': '>>',
     '-1': '<',
    '1': '>'
  }}
  />
</Calendar>
```

**Comment by radubrehar** [comment]
Just released `5.3.15` on npm with these improvements.

**Comment by israelidanny** [comment]
Can arrows still be svg after your modification?

**Comment by radubrehar** [comment]
Sure, they can be any valid jsx.

**Comment by israelidanny** [comment]
Great 👍 Thanks for the quick inclusion of PRs

**Comment by israelidanny** [comment]
I just tested, and the functionality doesn't work as expected on `5.3.15` 
The arrows aren't being overridden. 

Everything works great on my branch with the pull request I submitted, but somehow between then and the `navBarArrows` change, things broke?

**Comment by radubrehar** [comment]
Can you provide a code snippet?

**Comment by radubrehar** [comment]
Is this still reproducible for you?

**Comment by israelidanny** [comment]
Yeah, my pull request worked perfectly, but whatever changed in reality on master doesn't work at all... Try it yourself - arrows aren't being overriden

**Comment by israelidanny** [comment]
This is still broken. Can we just use my fix, that works perfectly? I'm not sure what's going on, but the thing that was actually derived doesn't work at all. Please test it and try to replace arrows.
